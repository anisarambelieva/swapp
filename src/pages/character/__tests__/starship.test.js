import { render } from "@testing-library/react";

import Starship from "../starship.js";

describe("<Starship />", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <Starship name="testname" imageSrc="test imageSrc" />
    );

    expect(container).toMatchSnapshot();
  });
});
