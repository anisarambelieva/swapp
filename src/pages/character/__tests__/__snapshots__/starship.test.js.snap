// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Starship /> should match snapshot 1`] = `
.c2 {
  font-size: 20px;
  font-weight: bold;
  margin: 0;
  color: #4bd5ee;
}

.c1 {
  background-color: white;
  border-radius: 0px 5px 5px 0px;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 {
  height: 100%;
  object-fit: cover;
  width: 100%;
  border-radius: 5px 0px 0px 5px;
}

<div>
  <div
    class="row"
    style="margin: 10px 0px 0px 0px; height: 70px;"
  >
    <div
      class="col-md-3"
      style="padding-right: 0px;"
    >
      <img
        class="c0"
        src="test imageSrc"
      />
    </div>
    <div
      class="col"
      style="padding-left: 0px;"
    >
      <div
        class="c1"
      >
        <h1
          class="c2"
        >
          testname
        </h1>
      </div>
    </div>
  </div>
</div>
`;
